conformance/base/admin_tier/experimental-egress-selector-rules.yaml

```diff
@@ -14,7 +14,7 @@ spec:
         matchLabels:
           conformance-house: gryffindor
   egress:
-  - name: "allow-egress-to-36363-on-nodes"
+  - name: "allow-egress-to-{{ index .Ports 0 }}-on-nodes"
```

Comment:
Maybe rename the files to `.yaml.in`? Not sure if that's normal/expected...

Reply:
I think this mostly makes sense if we end up generating a proper `yaml` file after applying the values, but that doesn't happen. We also don't use templating in all yamls, so some tests will have to use `yaml.in` as their manifest and others will use `yaml`.
So I figured it will bring more confusion that clarity